src/relaxng/materials.rnc

```diff
@@ -23,8 +23,15 @@ element materials {
       )
     }*  &
 
+    element macroscopic {
+      (element name { xsd:string { maxLength = "7" } } |
+        attribute name { xsd:string { maxLength = "7" } }) &
```

Comment:
Let's just remove the maxLength here. Looks like its 12 in the source code and probably will expand in the future.